hypha/apply/funds/views.py

```diff
@@ -90,6 +90,33 @@
 )
 
 
+class UpdateReviewerActions:
+    def set_status_after_reviewers_assigned(self, submission):
+        # Check if two internal reviewers have been selected.
+        internal_reviewers_count = submission.assigned.with_roles().count()
+        if internal_reviewers_count > 1:
```

Comment:
Reminder to self: Make use of has_all_reviewer_roles_assigned when https://github.com/OpenTechFund/hypha/pull/1927 is merged.